README.md

```diff
@@ -1 +1,56 @@
-# setup-opentofu WIP repo
+# setup-opentofu
+
+The `opentofu/setup-opentofu` action sets up OpenTofu CLI in your GitHub Actions workflow by:
+
+- Downloading the latest version of OpenTofu CLI and adding it to the `PATH`.
+- Configuring the [CLI configuration file](https://www.terraform.io/docs/commands/cli-config.html) with a Terraform Cloud/Enterprise hostname and API token.
```

Comment:
Hey! Thanks for the suggestion. I'd rather leave it as as for now. The rationale: we don't provide a cloud solution, hence it does make sense to point to the source. Moreover, their structure of the config stays intact. WDYT?

Reply:
Given that OpenTofu doesn't have a cloud/enterprise offering yet, that's fair enough. 

Would it be worth linking to OpenTofu's docs separately in parenthesis, since the Readme is the first page a user sees and it'd be preferable to highlight OpenTofu over Hashicorp's offerings, where possible?

Reply:
Thanks for your comment. You are right, the best tools which aim to enable engineers do link appropriate documentation indeed. I feel that linking tf enterprise cloud docs is appropriate here because the action configures it. IMO liking OpenTofu docs here will cause avoidable confusion at this moment. WDYT? cc: @eranelbaz @cube2222

Reply:
Implemented